frontend/libs/studio-components/src/components/StudioPageHeader/common.module.css

```diff
@@ -0,0 +1,40 @@
+.linkOrButton {
+  --studio-button-regular-dark: #393b51;
+  --studio-button-regular-medium: #3e4f62;
+  --studio-button-regular-light: #657281;
+
+  --studio-button-preview-dark: #0c6536;
+  --studio-button-preview-medium: #118849;
+  --studio-button-preview-light: #67b38a;
```

Comment:
Is there no FDS variables for these?

Reply:
Apparently not. Maybe we should centralize Studio-specific colour variables somewhere, but that'd be another issue. For now it's best to define them close to where they're used.